### DISCUSSION TITLE
新版进入游戏后怎么没有装备推荐了，看README说导入数据，好像也没看到导入按钮

### DISCUSSION THREAD

**Discussion by pycj** [cangzhang/ChampR#295] [General]

**Comment by cangzhang** [comment]
![image](https://github.com/cangzhang/champ-r/assets/1357073/0ecc0e35-bdb9-4de9-901e-495f2d0eeaeb)

**Comment by pycj** [comment]
我点击Apply之后进入游戏也没有
![屏幕截图 2023-09-02 185352](https://github.com/cangzhang/champ-r/assets/35558561/6ddf29ec-66d0-42a9-bb9d-df5b20384c6e)

**Reply by cangzhang** [reply]
看到 Applying Builds... 了吧，导入需要时间，等它消失。
点一次就够了，不用每次开游戏都点

**Reply by pycj** [reply]
一局打完了还没消失正常么

**Comment by pycj** [comment]
还是说导入需要翻墙？

**Comment by cangzhang** [comment]
如果是台服可以看这里 https://github.com/cangzhang/champ-r/discussions/292#discussioncomment-6808083

**Reply by pycj** [reply]
好了，谢谢
